app/models/queue_item.rb

```diff
@@ -0,0 +1,17 @@
+class QueueItem < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :video
+
+  delegate :category, to: :video
+  delegate :title, to: :video, prefix: :video
+
+  def rating
+    review = Review.where(user_id: user.id, video_id: video.id).first
```

Comment:
I would suggest to use `find_by` when searching for 1 record. like `review = Review.find_by(user_id: user.id, video_id: video.id)`